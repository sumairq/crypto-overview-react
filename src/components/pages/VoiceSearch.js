import React from 'react';

const VoiceSearch = () => (
  <div>VoiceSearch under construction....</div>
);

export default VoiceSearch;
