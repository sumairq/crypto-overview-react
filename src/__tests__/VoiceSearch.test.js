import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import VoiceSearch from '../components/pages/VoiceSearch';
import store from '../redux/configureStore';

test('matches voice search component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <VoiceSearch />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
