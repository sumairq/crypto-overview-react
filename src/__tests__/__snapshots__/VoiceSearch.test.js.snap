// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches voice search component snapshot 1`] = `
<div>
  VoiceSearch under construction....
</div>
`;
